crates/starknet-contract-class/src/lib.rs

```diff
@@ -190,7 +199,8 @@ mod tests {
 
         // We check only some of the attributes. Ideally we would serialize
         // and compare with original
-        assert_eq!(contract_class.abi, None);
+        // TODO: Add the real abi.
```

Comment:
Can we remove this TODO?

Reply:
Ok, do you mean creating an ABI like structure to match there, or just to delete the comment?

Reply:
Done